CONTRIBUTING.md

```diff
@@ -70,21 +70,21 @@ GitHub labels are used for organizing issues and providing context. You can fami
 
 ### Lifecycle
 
-There are four issue lifecycle labels:
+There are five issue lifecycle labels:
 
-- `0 - new`: Issues that are up for grabs.
-- `1 - assigned`: Issues that someone will work on soon.
-- `2 - in development`: Issues are currently being worked on.
-- `3 - installed`: Issues that have been merged to the `dev` branch and/or are ready for QA/QC.
-- `4 - verified`: Issues that have been tested, confirmed as mitigated, and are ready to close.
+- `1 - in design`: Issue that are actively undergoing design work.
+- `2 - ready for dev`: Issues that are fully designed, have acceptance criteria defined, and are ready for a developer to pick up.
+- `3 - in development`: Issues that are actively being worked on.
+- `4 - installed`: Issues that have been merged to the `dev` branch and/or are ready for QA/QC.
+- `5 - verified`: Issues that have been tested, confirmed as mitigated, and are ready to close.
 
 An issue can only have one of the lifecycle labels at any time. Please make sure to keep these up to date.
```

Comment:
Even though this is a bit out of scope for this PR, wondering if we should call out this out before listing out the lifecycle labels?